Migrate Streaming Analytics documentation to Amazon ECR (#2662)
Cumulocity-IoT/c8y-docs: content/streaming-analytics/epl-apps-bundle/basic-functionality.md

5. When you are ready to deploy to {{< product-c8y-iot >}}, upload the application as a microservice. For details, refer to [Managing microservices](/standard-tenant/ecosystem/#managing-microservices).

{{< c8y-admon-info >}}
After December 2024, the location of the Docker images has changed for all supported release trains.

> They are now available at *public.ecr.aws/apama* instead of at Docker Hub.

Reviewer: ```suggestion
They are now available at Amazon ECR Public Gallery instead of at Docker Hub.
```

Author: So, the reason I'm using this text in particular is because within the Dockerfile itself, users would now do 

`FROM public.ecr.aws/apama/apama-correlator`, instead of `FROM softwareag/apama`. The previous message here communicated a similar change, so I felt we could also do the same here.

Still, deferring to you on opinion.

Reviewer: If you look at the preview, you can see that this Info note is directly above the Important note, and the Important note then mentions the "public.ecr.aws/apama". As a counterpart to "Docker Hub", I would also use the normal, non-technical name in this Info note.

If you still use the images from the previous location, you must migrate them.
{{< /c8y-admon-info >}}

{{< c8y-admon-important >}}

> Apama 10.15.0 introduces several new container images provided via Amazon ECR and some of the existing container images have changed content.

Reviewer: Shouldn't this be 10.15.6 instead of 1015.0?

Author: Hm, I didn't actually properly parse that line before changing it, so will seek your opinion here

- The several new container images are 10.15.0,
- Amazon ECR is 10.15.6

What would be the best phrasing to convey that? Could be lazy and keep it as is, but I'm amenable to any option.

> Apama 10.15.0 introduces several new container images provided via Amazon ECR and some of the existing container images have changed content.

Reviewer: ```suggestion
Apama 10.15.0 introduces several new container images, and some of the existing container images have changed content. As of December 2024, these images are provided via Amazon ECR Public Gallery.
```

When building images for use as a {{< product-c8y-iot >}} microservice, this is now different to earlier releases.
You must now use the

> [public.ecr.aws/apama/apama-cumulocity-jre](https://gallery.ecr.aws/apama/apama-cumulocity-jre) image with the

Reviewer: ```suggestion
[apama/apama-cumulocity-jre](https://gallery.ecr.aws/apama/apama-cumulocity-jre) image with the
```

Reviewer: I'd remove the "public.ecr.aws" here so that it's the same name as in the Apama documentation at https://docbuilder1.eur.ad.sag/jenkins/job/pam/job/trunk-webhelp-migration-test_en/lastSuccessfulBuild/artifact/out/webhelp/index.html#page/pam-webhelp-4-markdown%2Fco-DepAndManApaApp_published_apama_container_images.html

Let's see how these changes will all turn out in the next preview.

Author: To be honest... not sure about Jack's changes there (based on https://github.softwareag.com/C2E/pam-doc/pull/89/files)

Reviewer: Hm, when I look at https://gallery.ecr.aws/apama/ , I cannot see the "public.ecr.aws" in the names on the cards. These all start with "apama/".

Author: Right, but if you click into any individual one (i.e. https://gallery.ecr.aws/apama/apama-builder), you'll see that the command they need to use is `public.ecr.aws/apama/builder`.

I've ran this by Matt, and created https://github.softwareag.com/C2E/pam-doc/pull/90 to fix up the pam-doc.

Reviewer: ok, ignore my suggestion then.

> [public.ecr.aws/apama/apama-cumulocity-builder](https://gallery.ecr.aws/apama/apama-cumulocity-builder) image as a builder image.

Reviewer: ```suggestion
[apama/apama-cumulocity-builder](https://gallery.ecr.aws/apama/apama-cumulocity-builder) image as a builder image.
```
